SpinCore_pp/power_helper.py

```diff
@@ -1,8 +1,74 @@
 from numpy import *
-
+import sympy as sp
+from scipy.integrate import quad
+from pyspecdata import *
```

Comment:
we don't do this inside a package.  The `from numpy import *` is actually problematic, as well.  It should be:
```
from numpy import r_
import numpy as np
```

Reply:
you can use pycharm to change the import statement and change all the uses, as well.

Reply:
do not `from pyspecdata import *`

Reply:
`import pyspecdata as psp`, etc.

Reply:
to clarify what I'm saying -- we've learned that `from ... import *` statements in general do not play well with sphinx, so do not use them inside a package.

Reply:
whoops didn't see that good catch